hoomd/md/pair.py

```diff
@@ -227,24 +227,31 @@ class LJ(_Pair):
         \end{eqnarray*}
 
     See :py:class:`_Pair` for details on how forces are calculated and the
-    available energy shifting and smoothing modes.  Use ``params`` dictionary
-    to set potential coefficients.
-
-    The following coefficients must be set per unique pair of particle types:
-
-    - :math:`\varepsilon` - *epsilon* (in energy units)
-    - :math:`\sigma` - *sigma* (in distance units)
-    - :math:`r_{\mathrm{cut}}` - *r_cut* (in distance units)
-      - *optional*: defaults to the global r_cut specified in the pair command
-    - :math:`r_{\mathrm{on}}`- *r_on* (in distance units)
-      - *optional*: defaults to the global r_on specified in the pair command
+    available energy shifting and smoothing modes.  Use `params` dictionary
+    to set potential coefficients. The coefficients must be set per 
+    unique pair of particle types.
+    
+    Attributes:
+        params (TypeParameter[``particle_types``, dict]):
+            The LJ potential parameters. The dictionary has the following keys:
+
+            * ``epsilon`` (``float``, **required**) - :math:`\varepsilon` - energy parameter (in energy units)
+            
+            * ``sigma`` (``float``, **required**) - :math:`\sigma` - particle size (in distance units)
+            
+            * ``r_cut`` (``float``, **optional**) - *r_cut* (in distance units) 
+              - *optional*: defaults to the global r_cut specified in the pair command
+              
+            * ``r_on`` (``float``, **optional**) - *r_on* (in distance units) 
+              - *optional*: defaults to the global r_on specified in the pair command
```

Comment:
`r_cut` and `r_on` are both separate attributes from the `params` attribute. They should be listed as individual attributes. This applies to other classes as well.

Reply:
All pair potentials have r_cut and r_on, you only need to document them once in `_Pair` under `Attributes`

These have a type of `(TypeParameter[tuple[``particle_types``, ``particle_types``], float]):`

Reply:
If the r_cut and r_on are attributes, should the example be something like `lj.r_cut[('A','A')] = 1.0` instead of `lj.params[('A', 'B')] = dict(epsilon=2.0, sigma=1.0, r_cut=3.0, r_on=2.0)`? I am a little confused with the sample example.

Reply:
Yes, `lj.r_cut[('A','A')] = 1.0` is correct. If there are examples that show `r_cut` and `r_on` as elements of the `param` dictionary, then they are incorrect. Likely they were not completely updated from the HOOMD v2 documentation.